src/blocks/kdeconnect.rs

```diff
@@ -34,13 +36,14 @@
 //! ```toml
 //! [[block]]
 //! block = "kdeconnect"
-//! format = " $icon {$bat_icon $bat_charge|}{ $notif_icon|} "
+//! format = " $icon {$bat_icon $bat_charge|}{ $notif_icon|} {$network_icon $network|}"
```

Comment:
I think we would want this to be:

```
//! format = " $icon {$bat_icon $bat_charge |}{$notif_icon |}{$network_icon $network |}"
```

Reply:
I shifted the space of network part before the `|` but I didn't touch the other parts that were there already. Should I modify those?

Reply:
Ah, yes. I think that was there was a mistake before you started your change.